op-e2e/actions/interop/dsl/utils.go

```diff
@@ -0,0 +1,24 @@
+package dsl
+
+import (
+	"context"
+
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func AssertAncestorDescendantRelationship(t require.TestingT, chain *Chain, ancestor, descendant eth.BlockID) bool {
+	assert.GreaterOrEqual(t, descendant.Number, ancestor.Number, "descendant block has a lower number than ancestor block")
+
+	current := descendant
+	result := true
+	for current.Number > ancestor.Number && current.Number > 0 {
+		// TODO: Update typing of t to use test-level context
```

Comment:
(unblocking comment): Note that every TODO must contain a issue number, and it was supposed to reported at CI.